tests/phpMyFAQ/Controller/Frontend/KeycloakAuthenticationControllerWebTest.php

```
@@ -0,0 +1,248 @@
+
+        $idToken = $this->signToken([
+            'iss' => 'https://sso.example.test/realms/phpmyfaq',
+            'aud' => 'phpmyfaq',
+            'azp' => 'phpmyfaq',
+            'exp' => time() + 300,
+        ]);
+        $expectedNonce = '';
+        $expectedVerifier = '';
+
+        $responseIndex = 0;
+        $httpClient = new MockHttpClient(function (string $method, string $url, array $options) use (&$responseIndex, &$idToken, &$expectedNonce, &$expectedVerifier): MockResponse {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$method'. (undefined)

(UnusedFormalParameter)